guides/line-management/README.md

```diff
@@ -0,0 +1,35 @@
+# Line management at Made Tech
+
+Line managers help others to learn and grow, deliver great results, and progress their careers.
+
+People who report to a line manager are called "direct reports".
+
+This guidance isn't complete yet. If you need help, ask:
+- the people team (**#team-people** on slack) 
+- the line manager community (**#supply-learning-line-managers** on slack)
+- Yasemin Ercan, our Head of People
+
+## What do I need to do as a line manager?
+
+### One-to-one meetings
+Hold regular one-to-one meetings, at least once a month.  
+
+### Probation
+Help your direct reports to pass their probation period.  
+Work with the Market Principal to extend or fail probations, if necessary.
+
```

Comment:
Throwing these out: 
- Provide a judgement on whether Made Tech is the right environment for their direct reports to succeed long-term
- Assert that their direct reports meet the minimum standards for their role

Reply:
I plan to sink these into the specific probation guidance - coming soon™